src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -3372,14 +3367,14 @@ trait Types
     }
   }
 
-  /** Precondition: params.nonEmpty.  (args.nonEmpty enforced structurally.)
+  /** Precondition: !params.isEmpty.  (args.nonEmpty enforced structurally.)
```

Comment:
Do we need also change in the comment? If so, shout it be coherent?

Reply:
I'd definitely shout it, Be coherent!